desktop/libs/indexer/src/indexer/indexers/rdbms.py

```diff
@@ -70,7 +70,7 @@ def _get_api(request):
   options = None
   query_server = None
   if file_format['rdbmsMode'] == 'customRdbms':
-    type = 'custom'
+    type = 'importer'
```

Comment:
`importer` feels too generic and out of place as type, only for custom external database connections via rdbms.

Should we keep `custom` and change accordingly? https://github.com/cloudera/hue/pull/3268/files#diff-833eafd5accb4467609950009daa4403e9cf8b8189076edb6589e172034554acR388

I feel this way we can skip this? What do you think?

```
It can be resolved by adding [[[custom]]] section in hue.ini

[[interpreters]]
  [[[custom]]]
    name=custom
    interface=rdbms

However, it doesn't look good to me. when you add [[[custom]]] section under the [[interpreters]] section, you can see custom item in the editor list even if it is useless.
So I decided to add importer type and use it. (For detailed information, Please check my commit.)
```

Reply:
> Nice work! I've added few review comments. Also, can you please elaborate more on the steps of how you tested it? A bit curious on what's the use-case here for external database.

@Harshg999 

Thank you for the review. it is really my pleasure to discuss the problem with you.
I will write my thought on your review comments shortly. 

### Elaboration of test process

The purpose of the use case is to migrate external database tables to hive tables.

I've provision my own hadoop cluster(hadoop + hue + oozie + sqoop + hive and etc..) and configured properly.

### Tested Use case scenario
#### 1. Importer with custom mode and jdbc driver
* Click 'Importer' on the side navigation menu.
* Select Type 'External Database'.
* Select Mode 'Custom'.
* Select Driver 'jdbc'.
* Fill in the 'url' text box.
* Fill in the 'JDBC Driver' text box. (write proper jdbc class name)
* Fill in the 'username' and 'password' for the database to connect.
* Click 'Test Connection' (Check 'snippet type custom is not configured' error message appears)
* Check connection status. 
* Choose Database and Tables.
* Click 'Next' button on the bottom and check it is possble to continue next page.
* Click 'Submit' button to execute oozie sqoop job.
* Check sqoop command is valid.
---
![image](https://user-images.githubusercontent.com/53476980/232182824-e25549d3-268e-44bd-905b-f06ee36b80f3.png)

---
<img width="1180" alt="image" src="https://user-images.githubusercontent.com/53476980/232182951-34a2dff0-7175-4304-825f-99e6c2a21e24.png">


#### 2. Importer with custom mode and MySQL driver
* Click 'Importer' on the side navigation menu.
* Select Type 'External Database'.
* Select Mode 'Custom'.
* Select Driver 'MySQL'.
* Fill in the 'hostname' and 'port' text box.
* Fill in the 'username' and 'password' for the database to connect.
* Click 'Test Connection' (Check 'snippet type custom is not configured' error message appears)
* Check connection status. 
* Choose Database and Tables.
* Click 'Next' button on the bottom and check it is possble to continue next page.
* Click 'Submit' button to execute oozie sqoop job.
* Check sqoop command is valid.
---
![image](https://user-images.githubusercontent.com/53476980/232183113-c9a6582b-e367-4914-a0d9-3b9426b8da78.png)

---
<img width="702" alt="image" src="https://user-images.githubusercontent.com/53476980/232183172-47f7cb15-09aa-40ce-b02d-6237c05b9cf1.png">

### Test Result
* I've check there are no more 'snippet type custom is not configured' error message in the use case scenarios.
* Test Connection is ok and Next page is available so that it is possible to submit oozie sqoop job.
* Created sqoop command is valid and migration was successful. (Table rows are not migrated, but this is not hue source code problem. The cause of the problem was HDFS permission)

Reply:
At first glance, 'custom' type feels too generic to me. 
But Analyzing the code, It is more reasonable to keep type name 'custom' rather than change it to 'importer'

I forgot the python script name is rdbms.py. so even if we keep type name as 'custom', it is not too generic and more understandable. 'custom' type will be recognized as customized rdbms in the whole code aspect.

I will rollback the type name and the dependant code too(https://github.com/cloudera/hue/pull/3268/files#diff-833eafd5accb4467609950009daa4403e9cf8b8189076edb6589e172034554acR388)